config/main.go

```diff
@@ -350,6 +357,27 @@ func setConfigModeBlockFields(f []string) {
 	}
 }
 
+// setSort sets the custom duration type
+func setSort(arg string) {
+	a := strings.ToLower(trimArg(arg))
+	if a == "" {
+		return
+	}
+
+	b, err := strconv.ParseBool(a)
+	if err != nil {
+		log.Printf("Sorting is disabled, invalid value: %v. Use 1 or true to enable, 0 or false to disable.", a)
+		return
+	}
+
+	if b {
+		Sort = true
+		log.Printf("Sorting is enabled")
+	} else {
+		log.Printf("Sorting is disabled")
+	}
```

Comment:
It looks like `setSort` can be called twice if both `yaml` config is set and `params` are passed to `config.Read()`. In other `set` functions, the `params` input overwrites the `yaml` config, but this isn't the case for `setSort` if `params.sort=false` and `yaml.sort=true`. I would just change this to match the logic of other `set` functions.
```suggestion
	Sort = b
	
	log.Printf("Sorting enabled: %v", Sort)
```

Reply:
Good point, thank you. Fixed.